package.json

```diff
@@ -59,6 +59,7 @@
     "lint-staged": "^10.0.8",
     "minimist": "^0.2.1",
     "npm-run-all": "^4.1.5",
+    "mxgraph-type-definitions": "^1.0.2",
```

Comment:
:question: do we need that in this PR? I had in mind that `mxgraph-type-definitions` would be introducedd in #301

Reply:
yes it is part of next PR, removing

Reply:
yes it is part of next PR, removing